docs/LTS-strategy.md

```diff
@@ -0,0 +1,38 @@
+# Express LTS Strategy
+
+## Goals
+
+* Maintain strict Node.js version support within a single Express major release
+* Provide good experience for users who want to keep their Express projects that up-to-date
+* Avoid stagnation and blocking innovation that is good for the end users
+* Ensure good ergonomics and developer experience for Express maintainers
+
+## Non-Goals
+
+* Only support the active LTS releases of Node.js
+* Provide timed cadence of Node.js version support dismissal
+* Support an extended set of Node.js versions to the detriment of all other factors. We aim to maintain a healthy balance between backwards compatibility and toolchain modernization
```

Comment:
@ljharb @wesleytodd @UlisesGascon Please review the updated wording

Reply:
How about this:
```suggestion
* Prioritizing "support older Node.js versions" over other factors. We aim to maintain a healthy balance between backwards compatibility and toolchain modernization
```

Reply:
sure, that works!